libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat

```diff
@@ -70,8 +72,8 @@ PC12 SPI3_MOSI SPI3
 PC11 SPI3_MISO SPI3
 PC10 SPI3_SCK SPI3
 
-PB5 LED OUTPUT HIGH GPIO(57)
-PB4 TIM3_CH1 TIM3 GPIO(56) ALARM
+PB5 LED OUTPUT HIGH GPIO(59)
```

Comment:
Please change this too
define HAL_GPIO_A_LED_PIN 57

Reply:
Will this conflict with PC9 definition above which is GPIO(57)? should a similar change be made to the omnibusF4 hwdef.dat

Reply:
It is not conflict. LED driver will just use gpio 57, which is now pwm pin. So led will not work. It should be define HAL_GPIO_A_LED_PIN 59 to match pin definition.
Omnibusf4 do not need this change, cause you did not change numbering.